source/extensions/omni.isaac.lab/omni/isaac/lab/sim/spawners/meshes/meshes_cfg.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) 2022-2024, The Isaac Lab Project Developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import MISSING
+from typing import Literal
+
+from omni.isaac.lab.sim.spawners import materials
+from omni.isaac.lab.sim.spawners.spawner_cfg import DeformableObjectSpawnerCfg
+from omni.isaac.lab.utils import configclass
+
+from . import meshes
+
+
+@configclass
+class MeshCfg(DeformableObjectSpawnerCfg):
```

Comment:
Let's fix inheritance or rename `MeshCfg` to `DeformableMeshCfg`

Reply:
Fixed now.